docs/reference/sites-filter.md

```diff
@@ -0,0 +1,95 @@
+# Sites filter on platform landing page
+
+This document describes how the theme can be configured to display a filterable
+set of exhibits on the platform landing page.
+
+
+## Features
+
+This feature meets the following requirements:
+
+* One-click access to multiple sites around Omeka S, directly from the platform
+  homepage.
+* Provides the ability to filter sites based on arbitrary categories, without
+  involvement by theme developers.
+* Enables interaction via keyboard, mouse, and screen readers.
+
+
+## Dependencies
+
+The filterable interface relies upon vanilla javascript and the jQuery library,
+which is already loaded by the theme. The interface also requires the platform
+to host an item set containing records for each exhibit that will appear on the
+homepage. These records are defined below.
+
+### Exhibit records
+
+The first, and most complex, part of this feature is the maintenance of an item
+set holding records for every exhibit on the platform. These records should have
+the following information:
+
+* Title (text - will be visible on screen)
+* Description (text - will be used as alt attribute for image)
+* URI (link - where users will be sent when clicking on the entry, should be the
+  exhibit homepage)
+* Subject (text, repeatable - used to populate filter options, so the set of
+  values should be curated across all exhibits)
+* Thumbnail (image - should be 200px square, and uploaded as the thumbnial via
+  the Advanced tab in the editor - not via the Media tab)
+
+These records should all be placed in a common Item Set, and kept in a Published
+state. Un-publishing them, or removing them from the Item Set, will prevent them
+from appearing in the filterable UI.
+
+### Theme configuation
+
+With the exhibit records created and placed in an Item Set, the theme can now be
+configured to display the filterable UI via two entries on the theme settings
+form:
+
+* Sites item set - this is the ID value of the Item Set which contains the
+  exhibit records.
+* Page slug that displays filterable site list - this should be the slug of the
+  platform landing page. For example, as of April 2024 the platform landing URL
+  is `https://digital-exhibits.libraries.mit.edu/s/home/page/welcome` - so this
+  value is `welcome`.
+
+When all the configuration above is complete, the feature should be operational.
+
+
+## Invocation
+
+The page template (`view/omeka/site/page/show.phtml`), at the bottom of the
+content area, loads the partial `common/sites-filter.phtml`. The filterable UI
+is entirely run from this partial.
+
+The partial checks for the two theme settings values. If both have been set, and
```

Comment:
`The partial checks for the two theme settings values (Sites item set and Page slug: see above for details).`